StyleCop.Analyzers/StyleCop.Analyzers/DocumentationRules/SA1600ElementsMustBeDocumented.cs

```diff
@@ -116,6 +121,10 @@ public void HandleMethodDeclaration(SyntaxNodeAnalysisContext context)
                 }
 
                 MethodDeclarationSyntax declaration = (MethodDeclarationSyntax)context.Node;
+                if (!(declaration.Parent is BaseTypeDeclarationSyntax))
```

Comment:
@sharwell That sounds reasonable. It doesn't really matter *what* we return because the code is broken anyway. We just need to not throw.